### ISSUE TITLE
Capture block formatting issue

### ISSUE THREAD

**Issue by marcinkrzeminski** [panoply/vscode-liquid#14]
Hi, 

I have the following piece

```
{% capture styles %}
  {% if fill %}
    fill: {{ fill }};
  {% endif %}
  {% if width %}
    width:  {{ width }}%;
  {% endif %}
```

Which is formatted into this:

```
{% capture styles %}
  {% if fill %}
    fill:
    {{ fill }};
  {% endif %}
  {% if width %}
    width:
    {{ width }}%;
  {% endif %}
```

Is there a way to make this work, except disabling formatting for capture block?

**Comment by panoply** [comment]
Not currently. The formatting parser used by this extension has trouble formatting `{% case %}` blocks, so for now we ignore them but with that said, it's probably better practise for you to avoid conditional statements within `capture` blocks.

**Comment by panoply** [comment]
@marcinkrzeminski follow issue [605](https://github.com/prettydiff/prettydiff/issues/605) in the PrettyDiff repository.

**Comment by panoply** [comment]
@marcinkrzeminski To keep you updated, this issue has be resolved within Sparser which is the PrettyDiff parser and now we are now waiting on a merge within PrettyDiff itself. The v2.0.0 branch of this extension with have the fix applied (plus a lot more additional features).

**Comment by panoply** [comment]
Closing this as it will be resolved in v2.0.0 to be released in the coming week. For now to avoid newlines use the following syntax:

```html
{% capture styles %}
  {% if fill %}
    {{ fill | prepend: 'fill:'  | append: ';'  }}
  {% endif %}
  {% if width %}
    {{ width | prepend: 'width:'  | append: '%;'  }}
  {% endif %}
{% endcapture %}
```
